spec/feature/user_story_spec.rb

```diff
@@ -0,0 +1,60 @@
+
+describe 'user stories' do
+  subject(:takeaway) { Takeaway.new({menu_klass: Menu.new, send_text_klass: SendText.new }) }
```

Comment:
Metrics/LineLength: Line is too long. [93/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/BracesAroundHashParameters: Redundant curly braces around a hash parameter.